luggage_aliases.module

```diff
@@ -40,8 +40,28 @@ function _luggage_aliases_people_node_form_submit($form, &$form_state) {
   }
 }
 
+/**
+ * Generate alias name.
+ */
 function _luggage_aliases_getTermName($form_state) {
-  $title = $form_state['values']['field_people_title'][LANGUAGE_NONE][0]['value'] . ' ' .  $form_state['values']['field_people_first_name'][LANGUAGE_NONE][0]['value'] . ' ' . $form_state['values']['field_people_middle_initial'][LANGUAGE_NONE][0]['value'] . ' ' . $form_state['values']['field_people_last_name'][LANGUAGE_NONE][0]['value'];
+  $title = '';
+    if ($form_state['values']['field_people_title'][LANGUAGE_NONE][0]['value'] !== '') {
+      $title_value = $form_state['values']['field_people_title'][LANGUAGE_NONE][0]['value'];
+      $title_label = $form_state['field']['field_people_title']['und']['field']['settings']['allowed_values'][$title_value];
+      $title .= $title_label . ' ';
```

Comment:
In the other if() blocks below the space is added in the previous line. It could be consistent in all the fields that we add the space to the line above and assign that to $title instead of creating a new variable. It's a minor style thing, so I'm also kinda meh about fixing this.